docs/README.md

```diff
@@ -1,4 +1,281 @@
+WinUI3 as a hosting frame lets you run modern web-based UX on all editions of Windows.
+
+* **[Modern Lifecycle helpers](https://github.com/microsoft/ProjectReunion/issues/9)** 
+help your app be power-sensitive and reactive to changes in 
+the runtime system's power management and user state.  They also help your app restart 
+after the user reboots, register your app so it can be restarted, and reduce update-related 
+reboots.
+
+* **[Startup Tasks](https://github.com/microsoft/ProjectReunion/issues/10)** 
+make your app come alive as the user logs in to reconnect, start working 
+while not over-using resources, or be ready for fast use.  User choice and power management 
+features let you reduce your impact on the critical logon path.
+
+* **Update Scan Integration** help keep your app up to date automatically while running at 
+the same time as other system maintenance tasks are happening.
+
+* **Access to user resources** even from AppContainer and isolated applications.  Your 
+AppContainer apps can access powerful Win32 technologies like the clipboard, inter-process 
+communication, and the Windows Shell Namespace with user consent through brokering.
+
+* **Modern Resource Tooling** so you can use the power of ResX/ResW in your Win32 applications 
+instead of MUI.
+
+
+# Future Direction
+We aim to grow Project Reunion to provide a modern, simple, and accessible version of much 
+of the power of the Windows Application Platform.  Working in the open with our development 
+community ensures we're doing the right things first to solve problems your apps are facing 
+on current and future versions of Windows.
+
+Some functionality will be delivered first as Project Reunion components. As we think about 
+evolving the platform it's important to make sure new functionality & features are available 
+to our developer community just as soon as they're ready to go.  Where there's newer 
+in-platform functionality to use Project Reunion will help you adopt that as soon as it's 
+widely available, without you having to retarget or rewrite � the same Project Reunion API 
+surface will continue to work.
+
+As your app uses more Project Reunion functionality it'll be ready for additional targets 
+such as packaging, identity, AppContainer, and other future Windows Editions. Depending on 
+Project Reunion APIs means we take on the work to keep these APIs working as Windows evolves 
+forward. Staying within the Project Reunion API surface ensures your app can address the 
+widest range of Windows editions and releases while using up-to-date functionality.
+
+# How we are doing this
+
+#### Working in the open
+Project Reunion relies on the power of open-source and the features of GitHub to deliver 
+code to you as fast as we can dream it up.  You should feel free to fork, create pull 
+requests, open issues, propose resolutions to issues.  You can see and influence all 
```

Comment:
"... open issues, and propose resolutions ..."

Reply:
Oxford comma :)  I will have the change in the next PR.